content-for/head-footer.html

```diff
@@ -1,5 +1,19 @@
 <style>
   .axe-violation {
-    border: 2px solid red !important;
+    background-image: repeating-linear-gradient(
+      45deg,
+      transparent,
+      transparent 0.4em,
+      hsla(69, 84%, 83%, 1.00) 0.4em,
+      hsla(69, 84%, 83%, 1.00) 0.8em,
+      hsla(166, 44%, 65%, 1.00) 0.8em,
+      hsla(166, 44%, 65%, 1.00) 1.2em,
+      hsla(214, 96%, 45%, 1.00) 1.2em,
+      hsla(214, 96%, 45%, 1.00) 1.6em
+    );
```

Comment:
Oh.... yep, that's absolutely true. I was only considering using the same class. Thanks for thinking outside of the box ~~model~~.